src/Set/SymfonyLevelSetList.php

```diff
@@ -6,6 +6,9 @@
 
 use Rector\Set\Contract\SetListInterface;
 
+/**
+ * @deprecated Use the constants of SymfonySetList instead.
+ */
```

Comment:
I currently deprecated it not sure when you planing the next release with breaking changes.